WalletWasabi/Helpers/Constants.cs

```diff
@@ -12,6 +12,7 @@ public static class Constants
 		public const string BackendMajorVersion = "3";
 
 		public const uint ProtocolVersionWitnessVersion = 70012;
+		public static readonly Version LegalDocsVersion = new Version(1, 0, 0, 0);
```

Comment:
This is very bad here. Client and Backend storing this version separately. There should not be any common file to store the version. The client stores its own local file version, Backend does the same. 
Also if I store this here, every time we update the docs we have to update and restart the server too. 

- Shall I put this version to somewhere else where it can be updated without restarting the backend? Like we did at RoundConfig.
- However, it is also not good if we have inconsistency among the repo files. Docs are in the repo so overwriting them on the backend and edit the version string manually without updating to the current master, could be a smelly method.